<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="UART_Set">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="URAT_Label_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口助手配置</string>
    </property>
   </widget>
   <widget class="QLabel" name="URAT_Label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="URAT_Label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="URAT_Label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>校验位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="URAT_Label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="URAT_Label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="OpenPortComName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="OpenBaudRateValue">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>640000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1152000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="OpenPortDataBit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="OpenPortCheckBit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="OpenPortStopBit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="ReceivedAndSendDataText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>440</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="OpenPortBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendDataBtn">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>220</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收数据：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ReceivedDataBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开始接收</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>循环发送周期：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="ToSendTextData">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearTextBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>清除界面</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED灯状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>按键状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>295</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AD值：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="AD_Value">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>陀螺仪与加速度计：</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>531</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="DrawCurve">
    <attribute name="title">
     <string>曲线图</string>
    </attribute>
    <widget class="QLabel" name="label_OSC">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Draw_3D">
    <attribute name="title">
     <string>三维图</string>
    </attribute>
    <widget class="QOpenGLWidget" name="openGLWidget_3D">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="show_2D" native="true">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextEdit" name="LED_State">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="KEY_State">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Gro_X_Edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>355</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>355</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gro_X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>385</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gro_Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Gro_Y_Edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>385</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>415</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gro_Z:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Gro_Z_Edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>415</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>445</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Acc_X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Acc_X_Edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>445</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>475</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Acc_Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>505</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Acc_Z</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Acc_Y_Edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>475</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Acc_Z_Edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>505</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Angle_Pitch">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>360</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>390</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Angle_Roll">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>450</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Angle_Yaw">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>480</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2D</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
